src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -53,6 +54,7 @@ async function validate_input_types(type, action, argv) {
         validate_no_extra_options(type, action, input_options_from_file, true);
         validate_options_type_by_value(input_options_with_data_from_file);
         validate_flags_combination(type, action, input_options_from_file);
+        validate_flags_value_combination(type, action, input_options_with_data);
```

Comment:
It is fine not to support the anonymous creation with `from_file` - just better to test that behavior (you can do it manually or in the unit tests).